.eslintrc-md.json

```diff
@@ -0,0 +1,15 @@
+{
+    "extends": "eslint-config-fluid",
+    "useEslintrc": false,
+    "env": {
+        "browser": true
+    },
+    "rules": {
+        "no-undef": "off",
+        "strict": "off",
+        "no-unused-vars": "off"
+    },
+    "plugins": [
+        "markdown"
+    ]
+}
```

Comment:
Suggestion (feel free to ignore) that we adopt a further linting plugin such as https://github.com/schorfES/grunt-lintspaces to enforce final newlines in this project and others

Reply:
Done and tested in depth with universal.  I ended up having to disable their indentation linting for Javascript comments and markdown, but otherwise it's hugely useful in catching syntax problems in previous unlinted files.